code/game/objects/items/implants/implant_misc.dm

```diff
@@ -34,10 +34,15 @@
 	. = ..()
 	uses--
 	to_chat(imp_in, "<span class='notice'>You feel a sudden surge of energy!</span>")
+	imp_in.SetSleeping(0)
 	imp_in.SetStun(0)
 	imp_in.SetKnockdown(0)
 	imp_in.SetUnconscious(0)
-	imp_in.adjustStaminaLoss(-75)
+	imp_in.adjustStaminaLoss(-200)
+	imp_in.updatehealth()
+	imp_in.update_stamina()
+	imp_in.stuttering = 0
+	imp_in.resting = 0
 	imp_in.lying = 0
```

Comment:
`lying` is the cached angle used for handling the rotation of the sprite after resting. This will break update_canmove()'s sprite rotation

Reply:
lying = 0 and update_canmove() was in the adrenal implant code before I even touched it tbh

Reply:
Also, without lying = 0, the rest button is very confused.